governance.md

```diff
@@ -18,88 +36,180 @@ The committee MUST:
-* Identify and document who is responsible for receiving vulnerability reports
-* Document process responsible parties go through to triage and determine veracity of vulnerability
-* Document process for facilitating fix, generating release update, and communicating vulnerability and fix to public
+Governance decisions, votes and questions should take place on the cdevents-governance@googlegroups.com mailing list.
+
+The governance committee decisions are taken by seeking consensus towards a motion from all its members. If the consensus cannot be reached, the motion may be altered or dropped.
+
+## Elections
+
+### Voter Eligibility
+
+Anyone who has at least 15 [contributions](./process/README.md#contributions) in the last 12 months. The [dashboard on cdevents.devstats.cd.foundation](https://cdevents.devstats.cd.foundation/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All) will show GitHub based contributions; contributions that are not GitHub based must be called out explicitly by the voter to confirm eligibility.
+
+### Candidate Eligibility
+
+Candidates themselves must be contributors who are eligible to vote, and must be nominated by contributors from at least two companies, which can include their own, and they can self-nominate.
+
+### Nominations Process
+
+Nominations should be sent to `cdevents-nominations@googlegroups.com`. The email
+should contain:
+
+- The nominee’s email address, github handle, company affiliation, and CDEvents
+  project(s) they contribute to
+- For each of two contributors nominating this individual:
+  - The company they work for
+  - Their github handles
+  - The CDEvents project(s) they contribute to
+
+Any nominee who accepts the nomination will be on the ballot.
+
+### Election Process
+
+Elections will be held using time-limited [Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) ranking on [CIVS](https://civs.cs.cornell.edu/) using the [Schulze method](https://en.wikipedia.org/wiki/Schulze_method). The top vote getters will be elected to the open seats. This is the same process used by the Kubernetes project. Voters must opt into the CIVS system at https://civs1.civs.us/cgi-bin/opt_in.pl or they won't receive an invitation to vote.
+
+Details about the schedule and logistics of the election will be announced in a timely manner by the election officers to eligible candidates and voters via the cdevents-dev@googlegroups.com mailing list.
+
+Example timeline:
+
+1. Before or during nominations, send an email to all eligible voters to notify them that they are eligible (allowing people time to reach out if they believe they are eligible but are not on our list)
+1. Prompt eligible voters to opt into CIVS at https://civs1.civs.us/cgi-bin/opt_in.pl so they will be able to vote
+1. 1 week for nominations (previously, starting on a Thursday until midnight UTC the next Wednesday)
```

Comment:
it might be difficult to extract the email addresses of all eligible candidates and voters from devstats and/or GitHub so it would be good if a voter registration form-like mechanism is put in place and documented.

Reply:
@e-backmark-ericsson sorry, copy paste issue

Reply:
@fdegir good idea, added